t04_refactored.py

```diff
@@ -135,9 +139,37 @@ def team_5_adv():
 
 
 def team_6_adv():
+    """
+
```

Comment:
Add the docstring information (what's this function do?)

Reply:
I've fixed this.